src/main/java/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,59 @@
+package calculator;
+
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+class StringAddCalculator {
+
+    private static final String DEFAULT_DELIMITERS = "[,;]";
+    private static final Pattern DELIMITER_REGEX_PATTERN = Pattern.compile("//(.)\n(.*)");
+    private static final int PATTERN_MATCHING_GROUP_FIRST_INDEX = 1;
+    private static final int PATTERN_MATCHING_GROUP_SECOND_INDEX = 2;
+
+
+    public static int run(String str) {
+        if (isEmptyOrNull(str)) {
+            return 0;
+        }
+        String[] splitStrings = split(str);
+        int[] numbers = parseStringToInt(splitStrings);
+        return sum(numbers);
```

Comment:
```suggestion
        return sum(parseStringToInt(split(str)));
```
로컬 변수 없이 위와 같이 메서드 체인으로 구현해 보는 것은 어떨까?